packages/yourloops/components/dashboard-widgets/device-usage-widget.tsx

```diff
@@ -131,7 +127,12 @@ const DeviceUsage = (props) => {
     }
   }
 
-  return <>
+  sortHistoryParametersByDate(history)
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏Given that these lines are the same as in `device-settings.tsx`, it would be nice to factorize this logic in a common method